src/udiskslinuxmanager.c

```diff
@@ -1211,11 +1213,15 @@ handle_resolve_device (UDisksManager         *object,
   g_variant_lookup (arg_devspec, "path", "&s", &devpath);
   g_variant_lookup (arg_devspec, "uuid", "&s", &devuuid);
   g_variant_lookup (arg_devspec, "label", "&s", &devlabel);
+  g_variant_lookup (arg_devspec, "partuuid", "&s", &partuuid);
+  g_variant_lookup (arg_devspec, "partlabel", "&s", &partlabel);
 
   blocks = get_block_objects (object, &num_blocks);
 
   for (blocks_p = blocks; blocks_p != NULL; blocks_p = blocks_p->next)
     {
+      found = FALSE;
```

Comment:
Oh I see. Yeah I added `found = FALSE;` since that would replicate what happened before, but yeah "only devices matching all values" is defunct currently. Will fix this.